spec/lib/embed/viewer/file_spec.rb

```diff
@@ -51,6 +51,13 @@
       html = Capybara.string(file_viewer.to_html)
       # visible false because we display:none the container until we've loaded the CSS.
       expect(html).to have_css '.sul-embed-file-list', visible: false
+      expect(html).to have_css 'a[download]', visible: false, text: '12.35 kB'
+    end
+    it 'with no file size' do
```

Comment:
Sounds fine with me to make the test for the general behavior with no-file size.